src/controllers/auth/login.js

```diff
@@ -0,0 +1,11 @@
+export const login = (res) => {
```

Comment:
i'm surprised this is working since the first argument provided in an Express route callback function is `req` and not `res`
```suggestion
export const login = (req, res, next) => {
```